modules/nf-core/mageck/test/meta.yml

```diff
@@ -0,0 +1,66 @@
+name: "mageck_test"
+description: write your description here
+keywords:
+  - sort
+  - rra
+  - CRISPR
+tools:
+  - "mageck":
+      description: "MAGeCK (Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout), an algorithm to process, QC, analyze and visualize CRISPR screening data."
+      homepage: "https://sourceforge.net/p/mageck/wiki/Home/#mle"
+      documentation: "https://sourceforge.net/p/mageck/wiki/Home/"
+      tool_dev_url: "https://bitbucket.org/liulab/mageck/src"
+      doi: "https://doi.org/10.1186/s13059-015-0843-6"
+      licence: "['BSD License']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - count_table:
+      type: file
+      description: |
+        Count table file.
+        Each line in the table should include
+        sgRNA name (1st column), target gene (2nd column)
+        and read counts in each sample.
+      pattern: "*.{txt,tsv}"
+  - contrast:
```

Comment:
There's no `contrast` input so this can be removed. If it's an args thing we don't currently document those.